src/style.css

```diff
@@ -1,3 +1,66 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
There is no need to reset margins and paddings for all selectors. You need to reset it only for the elements you need